include/Core/Users/UsersModule.php

```diff
@@ -88,6 +88,18 @@ protected function delete_users_from_query( $query, $options ) {
 				continue;
 			}
 
+			/**
+			 * Filter the user.
+			 *
+			 * @since 6.0.0
+			 *
+			 * @param array $options Processed options.
+			 * @param array $user    User Object.
+			 */
+			if ( apply_filters( 'bd_exclude_edd_customers', $options, $user ) ) {
```

Comment:
`if ( ! apply_filters( 'bd_can_delete_user', true, $user, $options, $this ) ) {`

Reply:
Fixed.